src/test/java/uk/gov/companieshouse/officerfiling/api/interceptor/RequestLoggingInterceptorTest.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.companieshouse.officerfiling.api.interceptor;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.logging.util.LogContextProperties;
+import org.springframework.web.servlet.ModelAndView;
+
+@ExtendWith(MockitoExtension.class)
+class RequestLoggingInterceptorTest {
+  private final static String TEST_REQUEST_PATH = "/";
+
+  @Mock
+  private Object handler;
+  @Mock
+  private ModelAndView modelAndView;
+  @Mock
+  private HttpServletRequest mockRequest;
+  @Mock
+  private HttpServletResponse mockResponse;
+  @Mock
+  private HttpSession requestSession;
+  private RequestLoggingInterceptor requestLoggingInterceptor;
+
+  @BeforeEach
+  void setUp() {
+    when(mockRequest.getSession()).thenReturn(requestSession);
+    when(mockRequest.getRequestURI()).thenReturn(TEST_REQUEST_PATH);
+    requestLoggingInterceptor = new RequestLoggingInterceptor();
+  }
+
+  @Test
+  void verifyRequestLoggingPreHandle() {
+    var response = requestLoggingInterceptor.preHandle(mockRequest, mockResponse, handler);
+    verify(mockRequest, times(1)).getSession();
```

Comment:
times(1) isn't required when you're only asserting on it being called a single time